### ISSUE TITLE
Not working with iOS 14 anymore

### ISSUE THREAD

**Issue by atoxx** [PhilipsHue/PhilipsHueSDK-iOS-OSX#126]
No matter what i do, i cannot connect to bridge on iOS 14 anymore, the bridge gests dicovered,, but after i press to connect, a "Connection lost" Error Message apears over and overr

**Comment by samuelclay** [comment]
Yep, I'm having the same problem. Their advice is to switch to the HTTP API, which I'll have to do, but that's a whole lot of work when this module was working fine before.

**Comment by atoxx** [comment]
How lame, philips, depricate sdk and no easy replacement, the whole link and connect part is a pain with the hue api, thx for nothing
